internal/v2/README.md

```diff
@@ -1 +1 @@
-This directory has implementation for S2Av2 go client.
+**This directory has implementation for S2Av2 go client**
```

Comment:
nit: s/implementation for S2Av2 go client/the implementation of the S2Av2's gRPC-Go client libraries.